src/modules/CVs/components/cv-details-form/cv-details-form.tsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect, useState } from "react";
+import { Navigate, useParams } from "react-router-dom";
+import { z } from "zod";
+import { RouteParams, routes } from "@/router";
+import { useAuthUser } from "@/services/auth-service";
+import { Flex } from "antd";
+import { Form } from "@/components/form";
+import { FormTextField } from "@/components/form-text-field";
+import { FormTextarea } from "@/components/form-textarea";
+import { useTranslation } from "react-i18next";
+import { FormSubmitButton } from "@/components/form-submit-button";
+import { useCvUpdate } from "@/modules/CVs/api";
+import { useNotificationContext } from "@/helpers/notification";
+import { useCvById } from "@/modules/CVs/api/get-cv-by-id-query";
+import styles from "./cv-details-form.module.css";
+
+const CvDetailsForm = () => {
+    const currentUser = useAuthUser();
+    if (!currentUser) {
+        return <Navigate to={routes.auth.root} replace />;
+    }
+    const { showNotification } = useNotificationContext();
+    const { t } = useTranslation();
+    const [isDisabled, setIsDisabled] = useState(false);
+
+    const cvDetailsSchema = () => {
+        return z.object({
+            name: z.string(),
+            education: z.string(),
+            description: z.string().min(6, {
+                message: t(`Description should be at least ${6} symbols long`),
+            }),
+        });
+    };
+
+    type cvDetailsSchemaType = z.infer<ReturnType<typeof cvDetailsSchema>>;
+
+    const { cvId } = useParams<RouteParams>();
+    if (!cvId) {
+        throw new Error("No cv id");
+    }
+
+    const { data: cvData, loading: loadingCv } = useCvById(cvId);
+    const [updateCv, { loading: updatingCv }] = useCvUpdate();
+
+    useEffect(() => {
+        if (cvData) {
+            setIsDisabled(currentUser.email !== cvData.cv.user.email);
+        }
+    }, [currentUser, cvData]);
+
+    const handleSubmit = async (values: cvDetailsSchemaType) => {
+        try {
+            return await updateCv({
+                variables: {
+                    cv: {
+                        cvId,
+                        name: values.name,
+                        education: values.education,
+                        description: values.description,
+                    },
+                },
+            });
+        } catch (error) {
```

Comment:
This error handling probably could be done in useMutation's onError option (since useCvUpdate is a hook and useNotificationContext can be used there too)